merkletree/src/rand_tools.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use rand::{CryptoRng, Rng, SeedableRng};
+use rand_chacha::ChaChaRng;
+use std::{num::ParseIntError, str::FromStr};
+
+#[derive(Debug, Copy, Clone)]
+pub struct Seed(pub u64);
```

Comment:
maybe a TODO stating that these are duplicates with types in tests_utils that some days should be resolved?

Reply:
Not really. This is OK. It doesn't have to be resolved. We may move this crate outside of mintlayer-core.